src/api-client/Qbert.js

```
@@ -145,19 +145,37 @@ class Qbert {
   }
 
   async getClusterPods (clusterId) {
-    return this.client.basicGet(`${await this.baseUrl()}/clusters/${clusterId}/k8sapi/api/v1/pods`)
+    try {
+      const data = await this.client.basicGet(`${await this.baseUrl()}/clusters/${clusterId}/k8sapi/api/v1/pods`)
+      return data.items.map(this.convertCluster(clusterId))
+    } catch (err) {
+      console.log(`Error getting cluster pods for clusterId: ${clusterId}`)
+      return []
```

Not sure we should be silencing the errors and returning an empty array.  This is going to lead to us not knowing there is an issue.

Use `console.error` for errors.